migrate/migrate.go

```diff
@@ -117,6 +138,10 @@ func Migrate(ctx context.Context, session *gocql.Session, dir string) error {
 		}
 	}
 
+	if err = AwaitSchemaAgreementOption.Check(ctx, session, awaitSchemaAgreementAfterMigrations); err != nil {
```

Comment:
I added the private `awaitSchemaAgreementAfterMigrations` option to unify the execution of AwaitSchemaAgreement on the gocql session (see line 40 for the check). I can also drop that and instead just put the three lines here again.

Reply:
L40 is ok with regards to that.